services/ip_manager/src/test/java/com/futurewei/alcor/ipmanager/IpManagerApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.futurewei.alcor.ipmanager;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class IpManagerApplicationTests {
```

Comment:
Let us plan to add more UTs as well.

Reply:
+1

Reply:
One general comment: We will need to verify the response as well. We could do it later.